packages/manager/modules/vps/src/vps/routing.js

```diff
@@ -76,22 +76,35 @@ export default /* @ngInject */ ($stateProvider) => {
       scrollToTop: () => () => {
         document.getElementById('vpsHeader').scrollIntoView();
       },
-      stateVps: /* @ngInject */ ($q, serviceName, OvhApiVps) =>
+      vpsVersion: /* @ngInject */ (serviceName, coreConfig, OvhApiVps) => {
+        if (coreConfig.isRegion('US')) {
+          return null; // version API is not required for US VPS
+        }
+
+        return OvhApiVps.v6().version({
+          serviceName,
+        }).$promise;
```

Comment:
**nitpcik:** Please use feature-flipping

Reply:
Hello @anooparveti, it's not a feature to enable/disable.
it's a legacy API that we have to remove at right moment.